packages/api/tests/api/admin/me/index.test.ts

```diff
@@ -0,0 +1,25 @@
+import express, { Request, Response } from 'express';
+import supertest from 'supertest';
+import { createMockHandler } from '../../../testUtils/expressHelpers/createMockHandler';
+import { getMock } from '../../../testUtils/getMock';
+import { get } from '../../../../src/api/admin/me/get';
+
+const mockGet = getMock(get);
+
+jest.mock('../../../../src/api/admin/me/get', () => ({
+  get: createMockHandler(),
+}));
```

Comment:
These are functionally equivalent but reading it like this is super strange because you're asking to get the mock before it's been mocked
```suggestion
jest.mock('../../../../src/api/admin/me/get', () => ({
  get: createMockHandler(),
}));

const mockGet = getMock(get);
```